db/schema.rb

```diff
@@ -44,5 +44,14 @@
     t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
   end
 
+  create_table "votes", force: :cascade do |t|
+    t.integer "user_id"
+    t.integer "votable_id"
+    t.string "votable_type"
+    t.boolean "vote_type"
```

Comment:
Any or all of this stuff required? It would be good to get `null: false` on anything that is (can be done in the migration). It would also be good tog et FK constraints an indexes on some of this, but we'll talk about that stuff later this week so it can wait until then.